robocorp-python-ls-core/src/robocorp_ls_core/libs/robocop_lib/robocop/config.py

```diff
@@ -13,6 +13,8 @@
 from robocop.exceptions import (
     ArgumentFileNotFoundError,
     CircularArgumentFileError,
+    VariableFileNotFoundError,
```

Comment:
The files under `robocorp_ls_core/libs/robocop_lib` are vendored files from `robocop` (so, they shouldn't really be used nor imported inside of the language server aside from just from its main public API to collect errors).

See the readme under `robocorp_ls_core/libs/robocop_lib` if an update is needed on how to update it (but if it has to be updated, it should be in its own PR).

Reply:
Thank you for your comments. I have corrected the other ones but I don't understand this one as I am not familiar with vendored files. Should this not be included here?

Reply:
Well, the files in `robocorp_ls_core/libs/robocop_lib` are actually from the 'https://pypi.org/project/robotframework-robocop/` project, so, we should not do any manual change on those files. 

If they need to be updated you need to follow the instructions from: https://github.com/robocorp/robotframework-lsp/blob/master/robocorp-python-ls-core/src/robocorp_ls_core/libs/robocop_lib/README.md (but as I see no updates in the `.dist-info` with a version update, I believe this wasn't done...).

Reply:
As a note, any changes under `https://github.com/robocorp/robotframework-lsp/blob/master/robocorp-python-ls-core/src/robocorp_ls_core/libs/robocop_lib/` must be reverted in this PR...

Reply:
If we need something from them, we should ask the project maintainers to update it and then we can update the vendored files.

Reply:
Thanks for your explanation. I reverted the changes.